src/app/modules/verification/verification.controller.ts

```diff
@@ -457,7 +295,7 @@ export const handleOtpVerification = [
 ] as ControllerHandler[]
 
 /**
- * Handler for resetting the verification state of a field.
+ * Handler for resetting the verification state of a field for POST /forms/:formId/fieldverifications/:id/fields/:fieldId/reset
```

Comment:
I think we can get rid of the `/reset` endpoints too actually, we don't actually use these...

Reply:
I think you're right. The function `resetVerifiedField` in the FE does not seem to be used anywhere. In that case we should do an e2e teardown of this endpoint. Rather than expanding the scope of this PR, I think we can do it in another!

Reply:
yes sir, let's goooo! merge when ready 👍

Reply:
Created linear issue FRM-1049 to track this!